yarn-project/end-to-end/src/e2e_2_pxes.test.ts

```diff
@@ -88,7 +88,7 @@ describe('e2e_2_pxes', () => {
 
     logger.info('L2 contract deployed');
 
-    return contract.instance;
+    return contract;
```

Comment:
This line causes a lot of changes in this test. I needed this change as I needed full TokenContract class instance in the `adds and fetches a nullified note` test. I think all the changes other than the mentioned test in this test file can be ignored as they were mechanical.